taca/cli.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
hmmm nope to be honest we have never worried about this. I guess it doesn't hurt but, any clear benefit?